aggregator/src/main/resources/logback-spring.xml

```diff
@@ -7,10 +7,10 @@
     <include resource="org/springframework/boot/logging/logback/base.xml" />  
  	<appender name="fileInfoLog"
 		class="ch.qos.logback.core.rolling.RollingFileAppender">
-		<File>/log/aggregator-info.log</File>
+		<File>aggregator-info.log</File>
```

Comment:
change back the settings, please don't submit code which only works in your env.